examples/static-site/Caddyfile

```diff
@@ -0,0 +1,5 @@
+0.0.0.0:8080 {
+        root /var/www/public
+        log stdout
+        gzip
```

Comment:
Could it be interesting to avoid compressing of images if they are located in a well-known place?